schema/workbook.json

```diff
@@ -916,6 +916,10 @@
                 "compositeBarSettings": {
                     "$ref": "#/definitions/compositeBarSettings",
                     "description": "used to store settings for composite bar"
+                },
+                "customColumnWidthSetting": {
+                    "type": "string",
+                    "description": "set a custom column width"
```

Comment:
add info about units here in the description?  or probably better an examples section with "15px", "10%", "5ch", "7fr", etc